### ISSUE TITLE
Youtube-Dl issues 

### ISSUE THREAD

**Issue by ghost** [eyaadh/megadlbot_oss#50]
Bot Doesn't Support www.youtube.com/videoid & youtu.be/videoid . It Supports Only m.youtube.com/videoid . Sending First two links gives Error & makes bot stucked Saying "Downloading Video to local" . & We need to Start Bot again .
Try to fix this issue .

& Second one it would be better if We Receive video Quality Choose button after sending Any links like below repo does :-

https://github.com/aryanvikash/Youtube-Downloader-Bot

**Comment by eyaadh** [comment]
In regards to the quality I did share you this at the telegram group too:
https://github.com/eyaadh/megadlbot_oss/issues/36
The project is not designed to do that as such selecting multiple qualities at download and this has been quite frankly explained on the issue shared as well the youtube series video for the module. 

Also in regards to the YouTube issue I did ask for more context as well at the group. 

https://t.me/megatronstatus/64431
https://t.me/megatronstatus/64475

Until the details are given I won't be able to help you i.e. until then will keep this issue closed.
